expr.go

```diff
@@ -77,7 +77,7 @@ func (e *comparisonExpr) isComplete() bool {
 
 // defaultValidateConvert will validate the comparison expr value, and then convert the
 // expr to its SQL equivalence.
-func defaultValidateConvert(columnName string, comparisonOp ComparisonOp, columnValue *string, validator validator, opt ...Option) (*WhereClause, error) {
+func defaultValidateConvert(columnName string, comparisonOp ComparisonOp, columnValue *string, validator validator, opts options) (*WhereClause, error) {
```

Comment:
perhaps don't change the signature.  Just call getOpts within the func which is
consistent with the rest of the code base.

Reply:
done